content/blog/2022/06/2022-6-15-svg-icon-migration.adoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "SVG icon migration (breaking changes!)"
+tags:
+- svg-migration
+- community
+- ux
+author: kmartens27
+description: >
+  Starting with Jenkins 2.333, Jenkins core has migrated from `GIF` and `PNG` icons to `SVG` icons.
+---
+
+== Summary
+
+Beginning with Jenkins v2.333 (released on June 15, 2022), Jenkins core has upgraded from `GIF` and `PNG` icons to `SVG` icons.
+Plugins that will be affected by this change are tracked in a link:https://docs.google.com/spreadsheets/d/1PxlgT11_uDyTzPch8zWn3PDxLUIAab21ILmJ17zCzBk/edit#gid=1308179162[spreadsheet] found in link:https://issues.jenkins.io/browse/JENKINS-68251[JENKINS-68251].
+Plugins that do not interact with icons are unaffected by this change.
+
+== Motivation
+
+Updating the Jenkins icons to these previously uploaded SVG icons provides users with a more consistent experience.
+SVG formatting also preserves the image quality during compression, ensuring that icons are displayed properly regardless of scale.
+
+For tracking purposes, link:https://issues.jenkins.io/browse/JENKINS-68251)[JENKINS-68251] is available and contains a link to a link:https://docs.google.com/spreadsheets/d/1PxlgT11_uDyTzPch8zWn3PDxLUIAab21ILmJ17zCzBk/edit#gid=1308179162[spreadsheet] listing plugins and their corresponding states to this update.
+*Make sure that you have used the Plugin Manager to update all plugins before and after updating Jenkins as failing to do so may result in errors.*
+
+
+== Reporting Issues
+
+In the spreadsheet, there is a table labeled “Abandoned plugin, to be discussed”

```

Comment:
Is there a title for the spreadsheet that can be added to make it more clear what you are referring to? And do you want the quoted text to be bold, instead? (Asking not suggesting, since this is a blog.)

Also, you are missing a period at the end of this sentence.